bangc-ops/mlu_op.h

```diff
@@ -10849,6 +10849,237 @@ mluOpPointsInBoxes(mluOpHandle_t handle,
                    const void *boxes,
                    const mluOpTensorDescriptor_t points_indices_desc,
                    void *points_indices);
+// Group:RoiAlign
+/*!
+ * @brief Computes the gradients of images \b grads_image based on the gradients \b grads and
+ * bounding boxes \b boxes to perform the backpropagation of ::mluOpRoiAlign operation. To use
+ * maximum pooling mode or average pooling mode in this operation, call ::mluOpRoiAlignBackward_v2.
+ *
+ * @param[in] handle
+ * Input. Handle to an MLUOP context that is used to manage MLU devices and queues in
+ * the roi_align_backward operation. For detailed information, see ::mluOpHandle_t.
+ * @param[in] grads_desc
+ * Input. The descriptor of the gradient tensor in the backpropagation process. For detailed
```

Comment:
The descriptor of the gradient tensor in the backpropagation process. > The descriptor of the tensor \b grads in the backpropagation process.